torch_geometric/data/summary.py

```diff
@@ -89,4 +115,24 @@ def __repr__(self) -> str:
             content.append(row)
         body = tabulate(content, headers='firstrow', tablefmt='psql')
 
+        if len(self.num_nodes_per_type) > 0 and \
+                len(self.num_edges_per_type) > 0:
+            content = [['']]
+            content[0] += [f'n: {node}' for node, _ in self.num_nodes_per_type]
+            content[0] += ['']
```

Comment:
It looks a little bit weird with that empty column.

I guess it could look better if you split node and edge stats into separete tables and then you could add a prefix saying sth like "Number of nodes per node type:", then those "n:" and "e:" also would be unnecessary.

It would be also a little bit narrower, what could be beneficial in case there are a lot of node and edge types.